docs/installing/adding_devices.md

```diff
@@ -64,29 +65,29 @@ The following instructions guide you through the process of installing the requi
 
 To install and configure your edge device, click the link that represents your edge device type:
 
-* [{{site.data.keyword.linux_bit_notm}} devices or virtual machines](#x86-machines)
-* [{{site.data.keyword.rhel}} 8.x devices or virtual machines](#rhel8)
-* [{{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines](#ppc64le-machines)
-* [{{site.data.keyword.linux_notm}} on ARM (32-bit)](#arm-32-bit); for example, Raspberry Pi running Raspberry Pi OS
-* [{{site.data.keyword.linux_notm}} on ARM (64-bit)](#arm-64-bit); for example, NVIDIA Jetson Nano, TX1, or TX2
-* [{{site.data.keyword.macOS_notm}}](#mac-os-x)
+- [{{site.data.keyword.linux_bit_notm}} devices or virtual machines](#x86-machines)
+- [{{site.data.keyword.rhel}} 8.x devices or virtual machines](#rhel8)
+- [{{site.data.keyword.linux_ppc64le_notm}} devices or virtual machines](#ppc64le-machines)
+- [{{site.data.keyword.linux_notm}} on ARM (32-bit)](#arm-32-bit); for example, Raspberry Pi running Raspberry Pi OS
+- [{{site.data.keyword.linux_notm}} on ARM (64-bit)](#arm-64-bit); for example, NVIDIA Jetson Nano, TX1, or TX2
+- [{{site.data.keyword.macOS_notm}}](#mac-os-x)
 
 ## {{site.data.keyword.linux_bit_notm}} devices or virtual machines
 {: #x86-machines}
 
 ### Hardware requirements
 {: #hard-req-x86}
 
-* 64-bit Intel&reg; or AMD device or virtual machine
-* An internet connection for your device (wired or wifi)
-* (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
+- 64-bit Intel&reg; or AMD device or virtual machine
+- An internet connection for your device (wired or wifi)
+- (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
 
 ### Procedure
 {: #proc-x86}
 
-Prepare your device by installing a Debian, {{site.data.keyword.rhel}} 7.x, or Ubuntu {{site.data.keyword.linux_notm}}. The instructions in this content are based on a device that uses Ubuntu 18.x.
+Prepare your device by installing a Debian, {{site.data.keyword.rhel}}, or Ubuntu {{site.data.keyword.linux_notm}}. The instructions in this content are based on a device that uses Ubuntu 18.x.
 
-Install the most recent version of Docker on your device. For more information, see [Install Docker](https://docs.docker.com/engine/install/ubuntu/).
+Install the most recent version of Docker on your device. For more information, see [Install Docker ](https://docs.docker.com/engine/install/ubuntu/){:target="_blank"}{: .externalLink}.
```

Comment:
Do we need to add a caveat here about RHEL 8.x and 9.x if they have Rodman 4.1 installed?  I assume these specific versions don't need Docker.

Reply:
Good point.  Fixed. Podman is also available on Ubuntu.  Docker is now optional.